2020-02-Spring/0_arrays_matrix/README.md

```diff
@@ -70,21 +70,33 @@ Go to [Solution](#s2)   [Top](#top)
 <!-- Don't remove -->
 <a name="p3"/>
 
-### 3. PROBLEM 3 TODO :bug:
+### 3. Word Search
 
-Source: TODO :bug:
+Source: [LeetCode](https://leetcode.com/problems/word-search/)
 
 #### Scenario
 
-Problem Statement TODO :bug:
+Given a 2D board and a word, find if the word exists in the grid.
+
+The word can be constructed from letters of sequentially adjacent cell, where "adjacent" cells are those horizontally or vertically neighboring. The same letter cell may not be used more than once.
 
 #### Example Input
 
-If the problem is simple enough, remove this section. TODO :bug:
+board =
+[
+  ['A','B','C','E'],
+  ['S','F','C','S'],
+  ['A','D','E','E']
+]
```

Comment:
Markdown's rendering behavior is not always what you'd expect. Since you were so careful about line breaks and indents here, I assume you want this to render the way it appears in a source file. The current version renders like this:

![rendering](https://user-images.githubusercontent.com/29528527/78927242-62ff6380-7a53-11ea-8682-8faa59eb1d63.png)

To get the monospaced appearance in the Markdown rendering, wrap it in code block backticks.

To view the Markdown rendering for the file, you can go into your PR and click the following:

![tabs](https://user-images.githubusercontent.com/29528527/78928805-1c5f3880-7a56-11ea-95c7-11beff0697e3.png)


````suggestion
```
board =
[
  ['A','B','C','E'],
  ['S','F','C','S'],
  ['A','D','E','E']
]
```
````

Reply:
Neat! :D TIL